eslint.config.mjs

```diff
@@ -0,0 +1,135 @@
+import importPlugin from 'eslint-plugin-import-x';
+import react from 'eslint-plugin-react';
+import globals from 'globals';
+import babelParser from '@babel/eslint-parser';
+import eslint from '@eslint/js';
+
+export default [
+  {
+    ignores: [
+      '**/.vscode/',
+      '.github/',
+      'media/',
+      'static/',
+      '**/coverage/',
+      'docs/_build/',
+      '**/package-lock.json',
+      '**/specs/',
+      '**/?venv/',
+      'translate/dist/',
+      // Jinja templates
+      'translate/public/translate.html',
+      '**/templates/**/*.html',
+      // Vendored code
+      'error_pages/css/blockrain.css',
+      'error_pages/js/',
+      'pontoon/base/static/css/boilerplate.css',
+      'pontoon/base/static/css/fontawesome-all.css',
+      'pontoon/base/static/css/nprogress.css',
+      'pontoon/base/static/js/lib/',
+    ],
+  },
+  eslint.configs.recommended,
+  react.configs.flat.recommended,
+  {
+    plugins: {
+      react,
+      import: importPlugin,
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.jest,
+        gettext: false,
```

Comment:
I had to check what `false` meant here, and it's a [deprecated synonym](https://eslint.org/docs/latest/use/configure/language-options#specifying-globals) for `'readonly'`, which would be clearer here.